samples/cpp/autofocus.cpp

```diff
@@ -244,60 +244,36 @@ static void showHelp(const char * pName, bool welcomeMsg)
 
 static bool parseArguments(int argc, char ** argv)
 {
-    int index;
-    GlobalArgs.deviceName = "Nikon";
-    GlobalArgs.output = NULL;
-    GlobalArgs.fps = DEFAULT_OUTPUT_FPS;
-    GlobalArgs.minimumFocusStep = 0;
+    cv::CommandLineParser parser(argc, argv, "{h help||}{o||}{f||}{m||}{d|0|}{v||}{@device|Nikon|}");
+    if (parser.has("help"))
+        return false;
```

Comment:
You haven't updated help message for this sample, it should show correct method for providing arguments, for example `-d=<focus-step>` or `-d=<INT>`